### ISSUE TITLE
Boots into stock boot animation

### ISSUE THREAD

**Issue by YoungBeta** [shalxmva/modxo#8]
Hello, I installed this modchip on a 1.0 xbox, and when I power it on, it goes into the stock boot animation. The hard drive and disk drove were not connected. What did I do wrong?
![20240413_211130](https://github.com/shalxmva/modxo/assets/97847973/73d1ac37-a332-4cfd-8d00-da770b2cfeb1)
![20240413_155418](https://github.com/shalxmva/modxo/assets/97847973/265a7b0e-8eb6-4280-94fc-f0454d035642)

**Comment by YoungBeta** [comment]
It also stops working after 20 seconds with the disk and hard drive plugged in

**Comment by MrKittenK** [comment]
Have you grounded D0?

**Comment by YoungBeta** [comment]
Yes, I have. I put the point under wire 2
![17130985675945989205331521112903](https://github.com/shalxmva/modxo/assets/97847973/003a9184-c903-40f4-9827-42763146c092)

**Comment by MrKittenK** [comment]
I mean is D0 grounded on the motherboard. 
It's the only other thing needed to do separately. 

There are vias and test points available on the upper and underside of the board. Plenty of guides online.

I've linked a pic as example
![1661955094_D0ona1 2 PNG 38681641200c83d2606409c3881ffc08](https://github.com/shalxmva/modxo/assets/91969885/cd0b955c-bdde-42a8-b41a-fb9d95fd3120)

**Comment by YoungBeta** [comment]
Thank you so much! I have a 1.0, so where would I put it on ground?

**Comment by MrKittenK** [comment]
![D0_Point_1 0_1 4](https://github.com/shalxmva/modxo/assets/91969885/80608015-845e-4805-b34c-f7fe64c9083a)
This shows the various points. 

I use the easier.to solder point on the bottom f the board, keep it short. 
If you want to swap to a modchip that can control the D0 line at some point then you'll have to unsolder and reattach to the new mod chip, so keep that in mind.

**Comment by YoungBeta** [comment]
Ok, thank you! I really appreciate your help.

**Comment by MrKittenK** [comment]
> Ok, thank you! I really appreciate your help.

You're very much welcome ✨
Always happy to help. 
Feel free to drop a message etc if you get stuck at all. I'm not the modXo creator, but I have some Xbox knowledge and love what Shalx has put together

**Comment by YoungBeta** [comment]
Hello, I ripped off the solder pad, but is there another point I could use?
![1713125203712403817348188168225](https://github.com/shalxmva/modxo/assets/97847973/fde4f545-b955-4cf2-a7ea-1a24d93e0a36)

**Comment by YoungBeta** [comment]
Alright, so I bought a new board and I'll try again. I'll provide a update when I get it!

**Comment by shalxmva** [comment]
For D0 grounding, you can use the motherboard screws, those have contact with ground pads.

**Comment by YoungBeta** [comment]
Thank you, I didn't know that! Also, how would I replace the trace on the old board? I know to scrape away the remaining trace, but where do I put the jumper to make it function again? I'm relatively new to soldering and the original xbox itself, so I would like to learn more about it. Thank you!

**Comment by MrKittenK** [comment]
There are alternative D0 spots. 
I always recommend flux, lots of flux, plus a temperature controlled soldering iron. If not temp controlled, keep to 15w and use 60/40 solder. 

I shall reply asap with alternatives.
1.0-1.1 uses one set.
1.2-14 uses another.
1.6 is already explained on the ModXo readme

**Comment by YoungBeta** [comment]
Thank you! Do I need to fix that trace for the console to boot, or after I ground an alternative point it will fix the FRAG that currently occurs? Again, I appreciate your help so much and you have been very helpful!

**Comment by MrKittenK** [comment]
I believe the trace will need to be fixed yes. 
I shall double check. 

Most welcome ✨

**Comment by YoungBeta** [comment]
Alright, I'll try to do that sometime.

**Comment by YoungBeta** [comment]
Alright, so I got the new board, and everything seemed to be working fine (got new hdd), and when I put it back together, it stopped working, so I bought a new Startech adapter, reflashed the pico with cerbios UDMA5
![20240421_184103_resized](https://github.com/shalxmva/modxo/assets/97847973/7297abe1-ac1d-4f64-8ab0-7a46c21ad9ae)
, and now I get error 21. I even reflashed with the hybrid bios again, but I still get error code 21. I used a different hard drive, and now it just hangs. The error picture is listed.
